apps/qubit/modules/menu/actions/quickLinksMenuComponent.class.php

```diff
@@ -28,12 +28,26 @@ class menuQuickLinksMenuComponent extends sfComponent
 {
   public function execute($request)
   {
-    // Get menu objects
-    $this->quickLinksMenu = QubitMenu::getById(QubitMenu::QUICK_LINKS_ID);
+    // Get menu
+    $quickLinksMenu = QubitMenu::getById(QubitMenu::QUICK_LINKS_ID);
 
-    if (!$this->quickLinksMenu instanceof QubitMenu)
+    if (!$quickLinksMenu instanceof QubitMenu)
     {
       return sfView::NONE;
```

Comment:
Nitpick: I believe `return sfView::NONE` is only relevant for sfAction objects and a bare `return` is sufficient to skip component output.

Reply:
Fixed!